README.md

```diff
@@ -77,9 +93,30 @@ If you had more features in the class and codes to extract landmarks or measurem
   16 |  Dorsal-most (upper) part of eye)  |  Top-most point of the eye mask defined by top boundary of the bbox  
   17 |  Ventral-most (lower) part of eye  |  Furthest bottom point of the eye mask defined by bottom boundary of the bbox  
   18 |  Center (centroid) of eye  |  Center of the eye mask  
+
+
+### Visualize
+
+This tool saves the segmented fish image with landmarks. 
+
+Here is an [example](Test_Data/INHS_FISH_000742_image_lm.png) of a visualization of the landmarks on the segmented image.
+
+
+### Metadata
+
+This is an input file. The file should be generated by [drexel](https://github.com/hdr-bgnn/drexel_metadata/tree/Thibault/gen_metadata_mini) and named basename_metadata.json.
```

Comment:
Again, we are referring to a branch in a repository, almost certainly not what we want to do. Presumably there's a better (more canonical) reference?